packages/react/src/components/DataTable/TableExpandRow.tsx

```diff
@@ -98,15 +107,15 @@ const TableExpandRow = React.forwardRef(
         [`${prefix}--parent-row`]: true,
         [`${prefix}--expandable-row`]: isExpanded,
         [`${prefix}--data-table--selected`]: isSelected,
-        [`${prefix}--data-table--slug-row`]: rowHasSlug,
+        [`${prefix}--data-table--ai-label-row`]: rowHasAILabel,
```

Comment:
I think we need both untill v12... people could be targetting these classes
```suggestion
        [`${prefix}--data-table--slug-row`]: slug && rowHasAILabel,
        [`${prefix}--data-table--ai-label-row`]: rowHasAILabel,
```

Reply:
Hey Ari, here Slug is just a constant name which I have changed to decorator in line no 78 of this file and added condition for both Slug or decorator. However, yes we can add both the classes [`${prefix}--data-table--slug-row`] and [`${prefix}--data-table--ai-label-row`] on same condition keeping in mind if any user is using the class name.